src/AutoRest.CSharp/build/CodeGeneration.targets

```diff
@@ -36,7 +36,7 @@
     <!-- Here we append the generate-test-project configuration to TypespecAdditionalOptions if it is specified -->
     <TypespecAdditionalOptions Condition="'$(GenerateTestProject)' != '' AND '$(TypespecAdditionalOptions)' != ''">$(TypespecAdditionalOptions)%3Bgenerate-test-project=true</TypespecAdditionalOptions>
     <TypespecAdditionalOptions Condition="'$(GenerateTestProject)' != '' AND '$(TypespecAdditionalOptions)' == ''">generate-test-project=true</TypespecAdditionalOptions>
-    <_TypespecAdditionalOptions Condition="'$(TypespecAdditionalOptions)' != ''">-TypespecAdditionalOptions "$(TypespecAdditionalOptions)"</_TypespecAdditionalOptions>
+    <_TypespecAdditionalOptions Condition="'$(TypespecAdditionalOptions)' != ''">--emitter-options "$(TypespecAdditionalOptions)"</_TypespecAdditionalOptions>
     <_LocalSpecRepo Condition="'$(LocalSpecRepo)' != ''">--local-spec-repo $(LocalSpecRepo)</_LocalSpecRepo>
     <_Debug Condition="'$(Debug)' == 'true'">"--debug"</_Debug>
```

Comment:
So this debug flag is only debugging the tsp-client, tsp-client will not pass this through to our emitter.
Therefore this is a false advertisement - this is never trying to debugging our generator.

Reply:
We may change it to debug emitter when $(Debug) is set.

<_Debug Condition="'$(Debug)' == 'true'">"--emitter-options debug=true"</_Debug>

Reply:
We may need to merge this with `TypespecAdditionalOptions`
Or we remove this to not provide this flag, and ask use to set in `TypespecAdditionalOptions`

Reply:
> we remove this to not provide this flag, and ask use to set in TypespecAdditionalOptions

This is the behavior we used to have.